samples/dash-if-reference-player/app/main.js

```diff
@@ -875,9 +886,43 @@ app.controller('DashController', ['$scope', '$window', 'sources', 'contributors'
 
     $scope.toggleCmcdEnabled = function () {
         $scope.player.updateSettings({
-            'streaming': {
-                'cmcd': {
-                    'enabled': $scope.cmcdEnabled
+            streaming: {
+                cmcd: {
+                    enabled: $scope.cmcdEnabled
+                }
+            }
+        });
+    };
+
+    $scope.toggleCmsdEnabled = function () {
+        $scope.player.updateSettings({
+            streaming: {
+                cmsd: {
+                    enabled: $scope.cmsdEnabled
+                }
+            }
+        });
+    };
+
+    $scope.toggleCmsdApplyMb = function () {
+        $scope.player.updateSettings({
+            streaming: {
+                cmsd: {
+                    abr: {
+                        applyMb: $scope.cmsdApplyMb
+                    }
+                }
+            }
+        });
+    };
+
+    $scope.updateCmsdEtpWeightRatio = function () {
+        $scope.player.updateSettings({
+            streaming: {
+                cmsd: {
+                    abr: {
+                        etpWeightRatio: parseInt($scope.cmsdEtpWeightRatio)
```

Comment:
Shouldnt this be a float value between 0 and 1? Or was the idea to limit it to more like an "on/off" switching? From `ThroughputHistory`:

````
                const etpWeightRatio = settings.get().streaming.cmsd.abr.etpWeightRatio;
                throughput = (throughput * (1 - etpWeightRatio)) + (etp * etpWeightRatio);
````

Reply:
The idea was between 0 and 1, I fix it